tests/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.24.2
 replace github.com/nginx/nginx-gateway-fabric/v2 => ../
 
 require (
-	github.com/nginx/nginx-gateway-fabric/v2 v2.1.1
```

Comment:
I'll rebase the base branch to main again so that we don't need these changes here